README.md

```diff
@@ -18,6 +18,7 @@
 
 | Module | Description |
 | ------ | ----------- |
+| [database](plugins/modules/database.ps1) | Creates and configures a database. |
```

Comment:
I promise this'll be unnecessary soon(ish) with the docs build stuff. I mean you could make it unnecessary today by using surge.sh as a host, but I figure you'll be more interested in GitHub pages as a docs host.. I just gotta find the time...